index.html

```diff
@@ -1592,16 +1554,36 @@ <h3 id="applying">
         <div class="issue" data-number="446"></div>
       </section>
     </section>
-    <section>
+    <section data-dfn-for="WebAppManifest" data-link-for="WebAppManifest">
       <h2>
-        Manifest and its members
+        <dfn>WebAppManifest</dfn> dictionary
       </h2>
+      <pre class="idl">
+          dictionary WebAppManifest {
+             TextDirectionType? dir;
```

Comment:
Because you don't have to have these declared in your manifest, they are optional. I was looking for optional in the web idl spec, but I see your point that they are all optional because of them being in a dict. This means that I will have to change most of the algorithms again though :)

Reply:
fixed

Reply:
We've [all made this mistake before](https://github.com/WICG/web-share/issues/19) :)